dpnp/backend/custom_kernels_elemwise.cpp

```diff
@@ -67,6 +67,50 @@
 
 #include <custom_1arg_2type_tbl.hpp>
 
+#define MACRO_CUSTOM_1ARG_2TYPES_MKL_OP(__name__, __operation__, __mkl_operation__)                                    \
```

Comment:
- Could you please add documentation for this macro (if is not exists).
- please do not use "MKL" in names